src/Microsoft.AspNetCore.Mvc.Razor/TagHelpers/BodyTagHelper.cs

```diff
@@ -18,10 +18,11 @@ public class BodyTagHelper : TagHelperComponentTagHelper
         /// <summary>
         /// Creates a new <see cref="BodyTagHelper"/>.
         /// </summary>
-        /// <param name="components">The list of <see cref="ITagHelperComponent"/>.</param>
+        /// <param name="manager">The <see cref="ITagHelperComponentManager"/> which contains the list
```

Comment:
"contains the list" -> "contains the collection" throughout this PR. (There's a few left after your comment updates.)